readme.md

```diff
@@ -15,55 +15,40 @@ Below are some potential recommended uses:
 
 ### A typical front-end project
 This config allows ES5 code and browser native functions. It will complain about ES6+ code and language features.
+It also automatically applies the Vue plugin and Vue-specific rules to `.vue` files.
```

Comment:
Are they Vue rules a generic Wikimedia style guide, or are they specific to how Vue is used in MediaWiki?

Reply:
They are generic. The MW-specific rules were already in mediawiki.json, and I kept them there. A few things were specific to ES5 Vue code, and I kept those in vue-es5